src/main/java/seedu/address/logic/commands/AppointmentCommand.java

```diff
@@ -66,8 +68,18 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
         Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = new Person(personToEdit.getName(), personToEdit.getPhone(),
-                 personToEdit.getEmail(), personToEdit.getTags(), appointment, personToEdit.getProperty());
+        Person editedPerson;
+
+        if (personToEdit instanceof Buyer buyer) {
```

Comment:
Can assign person.getName() to a variable. This variable can be used in the constructor for both Buyer and Seller. Do this to avoid code reuse and make use of polymorphism.

Reply:
Similar to HaoZhen's comment in AddPropertyCommand